lib/cosmos/tools/cmd_tlm_server/gui/interfaces_tab.rb

```diff
@@ -154,7 +154,7 @@ def create_button(name, interface, interface_name)
       button = Qt::PushButton.new(button_text)
       if name == ROUTERS
         button.connect(SIGNAL('clicked()')) do
-          if interface.thread
+          if CmdTlmServer.routers.all[interface_name].thread
```

Comment:
Does this type of issue show up anywhere else? Just wanted to know if we should be referencing CmdTlmServer variables in other places.